### PR TITLE
Pagetree without admin style

### PR DESCRIPTION
**Pull request by vxsx**
fixes styles of the pagetree on django only installs

### CODE REVIEW

File: cms/static/cms/sass/components/pagetree/_node-state.scss [hunk 1]
```diff
@@ -7,14 +7,15 @@
...
+    vertical-align: top;
 }
 .cms-pagetree-node-state {
     &-empty {
         border-color: $gray-light;
         background-color: $white;
     }
     &-unpublished,
-    &-unpublishedparent {
+    &-unpublished-parent {
```
Comment (Brankub) [inline cms/static/cms/sass/components/pagetree/_node-state.scss]: Do not see any BE or FE changes related to this. :-| It was done before?

**Comment by Brankub** [comment]
overall LGTM

File: cms/static/cms/sass/components/pagetree/_node-state.scss (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/sass/components/pagetree/_node-state.scss]: that is something that was overlooked way before

Comment (Brankub) [inline cms/static/cms/sass/components/pagetree/_node-state.scss]: 👍

**Pull request merged**